proparse/src/main/java/org/prorefactor/refactor/RefactorSession.java

```diff
@@ -74,19 +74,35 @@ public Schema getSchema() {
   }
 
   /**
-   * Get the listing file name, makes sure the directory exists.
-   */
-  public String getListingFileName() {
-    return "listingfile.txt";
-  }
-
-  /**
    * Returns the Settings for the currently loaded project
    */
   public IProparseSettings getProparseSettings() {
     return proparseSettings;
   }
 
+  public File findFile3(String fileName) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) The Cyclomatic Complexity of this method "findFile3" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)